models.rb

```diff
@@ -4,21 +4,22 @@
 
 def read_all_memos
   original_data = File.read('db.json')
-  memos = JSON.parse(original_data, symbolize_names: true)
-  memos || []
+  JSON.parse(original_data, symbolize_names: true)
 end
 
 def find_memo(id)
   memos = read_all_memos
   memos.find { |memo| memo[:id] == id }
 end
 
+def overwrite_memos(memos)
```

Comment:
```suggestion
def write_memos(memos)
```

プログラミングではファイルの書き込みはシンプルにwriteと表現することが多いですね。

Reply:
考えすぎました💦`write_memos`に変更しました。https://github.com/SuzukaHori/sinatra_practice/pull/1/commits/1ef3079c6d631c696919f476256ee92fb2c25968